.vib/prestashop/vib-publish.json

```diff
@@ -17,6 +17,19 @@
       ]
     },
     "verify": {
+      "context": {
+        "resources": {
+          "url": "{SHA_ARCHIVE}",
+          "path": "/bitnami/prestashop"
+        },
+        "runtime_parameters": "cHJlc3Rhc2hvcFVzZXJuYW1lOiB1c2VyCnByZXN0YXNob3BQYXNzd29yZDogQ29tcGxpY2F0ZWRQYXNzd29yZDEyMyE0CnByZXN0YXNob3BIb3N0OiB2bXdhcmUtcHJlc3Rhc2hvcC5teQpzZXJ2aWNlOgogIHR5cGU6IExvYWRCYWxhbmNlcgogIHBvcnRzOgogICAgaHR0cDogODA=",
+        "target_platform": {
+          "target_platform_id": "{VIB_ENV_TARGET_PLATFORM}",
+          "size": {
+            "name": "S4"
```

Comment:
Do you think we could increase the cluster size of this chart? You mentioned it's really slow, would that help?

Reply:
I have tried with this size and got 3 consecutive green executions, so there is not a strong reason to use a bigger instance.

I think the slowness came from using DockerDestop's K8s cluster.

Reply:
Good to know, in that case, it's better to keep it small.